config/locales/en/general.yml

```diff
@@ -351,6 +363,16 @@ en:
       title: Proposals
       top: Top weekly
       top_link_proposals: The most supported proposals by category
+      section_header:
+        icon_alt: Proposals icon
+        title: Proposals
+        description: Make a citizen proposal. If get the enough supports and go to vote, you can get all the citizen decide how they want it to be our city.
```

Comment:
"our city" => "their city" ? That's more a question about the voice of the text, seems to me like the help is written from the perspective of an external person, not from "us/we/our" side right?

Reply:
Hum, I don't know how to answer that, I don't know if we want to write the text in a second or third person. As the platform will be part of a City Council I thought it could make more sense to make the _writer_ part of it :)